assets/js/util.js

```diff
@@ -19,6 +19,31 @@
 		return false;
 	});
 
+	  /**
+   * Intial loading of Order form based on product type
+   */
+  $(document).ready(function () {
```

Comment:
why do we need this if each type of order form is independent of each other? 

If this is to make the form dynamic so it can be used on all 4 categories of products, than we need to include pies and cookies. But that has its own trade off, because then the Formspree will have a bunch of column that are empty for each order
For example if you order cake, then there'll be cupcakeQuantity, pieSize, cookieQuantity columns blank, and they can look clutter to the owner.